Formula/inspectrum.rb

```diff
@@ -0,0 +1,22 @@
+class Inspectrum < Formula
+  desc "Offline radio signal analyser"
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.2.tar.gz"
+  sha256 "50b7db9b86208f414c387700a358eb58364094f3e8a4985f586f4f815645898a"
+  head "http://github.com/miek/inspectrum.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "qt"
+  depends_on "fftw"
+  depends_on "liquid-dsp"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system bin/"inspectrum", "-h"
```

Comment:
Can we actually get it to do something on the command-line?

Reply:
No, running the program just opens a GUI, except for the -h switch, which just prints the help and exits.

Reply:
Then can we at least match some output from `inspectrum -h`, like a version number?